### ISSUE TITLE
callout looks weird

### ISSUE THREAD

**Issue by Mollyoyo** [hydescarf/Obsidian-Theme-Mado-11#16]
please fix this, thanks :)
This is a really beautiful theme, thank you for your efforts (*^_^*)

![image](https://user-images.githubusercontent.com/30024196/202722456-383ee20c-b62c-4179-ba0d-40fa88b29799.png)

**Comment by hydescarf** [comment]
Thank you :)
Actually, this is the default callout design:
![image](https://user-images.githubusercontent.com/56707567/202818407-97a9a68b-8319-47c4-bb4c-504866a8253d.png)
Are you perhaps, using the incorrect markdown format for that? Or maybe you have other css snippets enabled that affected that?

**Comment by Mollyoyo** [comment]
> Thank you :) Actually, this is the default callout design: ![image](https://user-images.githubusercontent.com/56707567/202818407-97a9a68b-8319-47c4-bb4c-504866a8253d.png) Are you perhaps, using the incorrect markdown format for that? Or maybe you have other css snippets enabled that affected that?

I did not use any css snippets and the callouts code was copied from the obsidian help page. Now that I have installed this plugin: Admonition, the callouts work fine. So, maybe there is some bug in my obsidian. Thanks for your reply!

**Comment by hydescarf** [comment]
You shouldn't really need a plugin to fix that. I think the code you copied was just slightly mispositioned like something below:
![image](https://user-images.githubusercontent.com/56707567/202945513-4d2e7848-21aa-4c33-84f0-b76c3ecbd865.png)

If you'd like, you can screenshot the markdown and I'll take a look at it.

**Comment by Mollyoyo** [comment]
![image](https://user-images.githubusercontent.com/30024196/203290593-64860b47-4f63-4e97-8e49-e1fb8025551c.png)
![image](https://user-images.githubusercontent.com/30024196/203286391-b54fd00f-8a35-4edb-8444-fd72246aaa49.png)

At first I thought I had written the wrong code, so I went to obsidian help to copy the code, and it was the same. That's why I thought it was a bug.
I've just tested it after removing the admonition and it doesn't have that problem anymore.
